broker-daemon/broker-rpc/broker-rpc-server.js

```diff
@@ -79,21 +52,22 @@ class BrokerRPCServer {
     this.pubKeyPath = pubKeyPath
     this.privKeyPath = privKeyPath
     this.disableAuth = disableAuth
+    this.auth = verifyBasicAuth(rpcUser, rpcPass, disableAuth)
```

Comment:
[nit] I wouldn't call this "verify" since that seems to indicate that it is checking auth in this call. Maybe something like `basicAuth` or `createBasicAuth`?

Reply:
I would prefer `createBasicAuth`